modules/apps/message-boards/message-boards-web/src/main/java/com/liferay/message/boards/web/internal/display/context/EditMBDisplayContext.java

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.message.boards.web.internal.display.context;
+
+import com.liferay.message.boards.model.MBMessage;
+import com.liferay.portal.kernel.bean.BeanParamUtil;
+import com.liferay.portal.kernel.portlet.LiferayPortletRequest;
+import com.liferay.portal.kernel.portlet.LiferayPortletResponse;
+import com.liferay.portal.kernel.portlet.LiferayWindowState;
+import com.liferay.portal.kernel.portlet.PortletURLUtil;
+import com.liferay.portal.kernel.util.Constants;
+import com.liferay.portal.kernel.util.HashMapBuilder;
+import com.liferay.portal.kernel.workflow.WorkflowConstants;
+
+import java.util.Map;
+
+import javax.portlet.PortletURL;
+import javax.portlet.ResourceURL;
+
+/**
+ * @author Ambrín Chaudhary
+ */
+public class EditMBDisplayContext {
+
+	public EditMBDisplayContext(
```

Comment:
Rename it to `MBEditMessageDisplayContext` to keep it consistent with the others.